packages/common/src/components/DeactivatedAccount.tsx

```diff
@@ -0,0 +1,116 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import styled from "styled-components";
+import IconButton from "./IconButton";
+import { IconArrowRight } from "hds-react";
+import { fontBold, fontRegular } from "../common/typography";
+
+const ErrorContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  max-width: var(--container-width-xl);
+  margin: 0 auto;
+  gap: 42px;
+  padding-block: 42px;
+  padding-inline: var(--spacing-s);
+  --spacing-hz: var(--spacing-s);
+  img {
+    width: 185px;
+    height: auto;
+    margin: 0 auto;
+  }
+  @media (min-width: 768px) {
+    flex-direction: row;
+    padding-inline: var(--spacing-m);
+    --spacing-hz: var(--spacing-m);
+    img {
+      display: flex;
+      margin: 0;
+      margin-inline: auto;
+      width: auto;
+      max-height: 418px;
+      max-width: 350px;
+      flex-grow: 1;
+    }
+  }
+`;
+
+const TextContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 42px;
+  justify-content: center;
+  @media (min-width: 768px) {
+    order: -1;
+  }
+`;
+
+const Heading = styled.h1`
+  margin: 0;
+  line-height: 1;
+  font-size: 40px;
+  ${fontRegular}
+`;
+
+const Body = styled.p`
+  margin: 0;
+`;
+
+const Email = styled.a`
+  margin: 8px 0 0;
+  ${fontBold}
+`;
+
+// NOTE: This is a copy of the Footer component from the common package
+const constructFeedbackUrl = (
+  feedbackUrl: string,
+  i18n: { language: string }
+) => {
+  try {
+    const url = new URL(feedbackUrl);
+    url.searchParams.set("lang", i18n.language);
+    return url.toString();
+  } catch (e) {
+    return null;
+  }
+};
+
+const DeactivatedAccount = ({
+  feedbackUrl,
+  imgSrc,
+}: {
+  feedbackUrl: string;
+  imgSrc: string;
+}) => {
+  console.log(feedbackUrl);
+  const { t, i18n } = useTranslation();
+  return (
+    <ErrorContainer>
+      <img
```

Comment:
`next/image` still not working after you fixed the img source path?

Reply:
Yeah, now it seems to work on both sides => using `next/image` instead of `img`.